packages/core/src/router/ClientRouter.js

```diff
@@ -199,6 +205,16 @@ export default class ClientRouter extends AbstractRouter {
   handleError(params, options = {}, locals = {}) {
     if ($Debug) {
       console.error(params.error);
+
+      // Show error overlay
+      if (window.__IMA_HMR) {
+        window.__IMA_HMR.emit('error', {
+          error: params.error,
+          type: 'runtime',
+        });
+
+        return params.error;
```

Comment:
keep return type `{ content: null, status: options.httpStatus || 500, error: params.error}`

Reply:
Fixed in https://github.com/seznam/ima/pull/175/commits/bf8f7f628b4a77abff8a89306fbd1adf6bfb863a